Dockerfile

```diff
@@ -29,9 +29,13 @@ ENV MESOS_MARATHON_HOST http://servicehost:8080
 # Copy in Nginx and Supervisor config
 COPY docker/nginx.conf /etc/nginx/nginx.conf
 COPY docker/mc2.nginx.conf /etc/nginx/conf.d/
-COPY docker/supervisord.conf /etc/supervisord.conf
+COPY docker/supervisord.conf /etc/supervisor/supervisord.conf
 COPY docker/mc2.supervisor.conf /etc/supervisor/conf.d/
 
+# Send Nginx access and error logs to stdout/stderr
+RUN ln -sf /dev/stdout /var/log/nginx/access.log \
+    && ln -sf /dev/stderr /var/log/nginx/error.log
+
```

Comment:
I copied this from the official Nginx Docker image but I guess they might've just been doing this because they don't change the config file...